### PR TITLE
Fix code template language.

### PR DESCRIPTION
**Pull request by icycodes**

### CODE REVIEW

File: packages/quick-question/pages/api/search.ts [hunk 1]
```diff
@@ -31,11 +31,12 @@ export default async function handler(
...
       const formattedResults = queryResult.map(async (result: any[]) => {
         const code = result[0].pageContent;
-        const prompt = CodeTemplate.format({ query, code });
+        const language = result[0].metadata.language;
+        const prompt = CodeTemplate.format({ language, query, code });
         return {
           pageContent: code,
           metadata: {
-            language: result[0].metadata.language,
+            language: language,
```
Comment (wsxiaoys) [inline packages/quick-question/pages/api/search.ts:39]: what string will be here when file is typescript / javascript / tsx？

Comment (icycodes) [reply packages/quick-question/pages/api/search.ts:39]: `'tsx'` for typescript / javascript / tsx files

Comment (wsxiaoys) [reply packages/quick-question/pages/api/search.ts:39]: that would not very useful for language model （llm treat these string differently）

Let's make sure each language has its formal name here.

An easy solution would be just assign language names in indexer for different  extensions.

Comment (icycodes) [reply packages/quick-question/pages/api/search.ts:39]: Sounds good

**Pull request merged**